templates/service-template.yml

```diff
@@ -381,6 +381,24 @@ objects:
         targetPort: 8080
         name: metrics
 
+  - kind: Service
```

Comment:
is it possible to create it only if the grpc is enabled? it seems cannot add if else logic in template file.